buildspec.yml

```diff
@@ -0,0 +1,31 @@
+version: 0.2
```

Comment:
Do we want to add some description of how to use this to our `DEVELOPMENT.md`?

Reply:
Including required env variables probably.

Reply:
Added to development.md